src/views/hardware_view.rs

```diff
@@ -574,14 +575,16 @@ fn filter_options(
         .cloned()
         .collect();
 
-    if !config_options.contains(&PinFunction::None) && selected_function.is_some() && selected_function != Some(PinFunction::None) {
+    if !config_options.contains(&PinFunction::None)
+        && selected_function.is_some()
+        && selected_function != Some(PinFunction::None)
+    {
```

Comment:
just formatting changes, right?

Has something changed regarding formatting settings again?
Was it me?

Reply:
I didn't commit any changes in hardware_view